blog/2015/12/escience-dssg.html

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+root: ../../..
+author: [Ariel Rokem, Micaela Parker, Sarah Stone, Anissa Tanweer]
```

Comment:
Right now we need "Ariel Rokem, Micaela Parker, Sarah Stone, and Anissa Tanweer" - I'll fix during merge.  (We're going to do a proper list-based author layout in the [new website](https://github.com/swcarpentry/website).)